### PR TITLE
Modify the policy overview

### PR DESCRIPTION
**Pull request by menakaj**
## Automation tests
 - Unit tests 
   > Code coverage information
 - Integration tests
   > Details about the test cases and coverage

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes/no
 - Ran FindSecurityBugs plugin and verified report? yes/no
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes/no

### CODE REVIEW

File: packages.json [hunk 1]
```diff
@@ -1,38 +1,38 @@
 [
```
Comment (pubudu91) [inline packages.json:1]: This file shouldn't be modified. These versions < Ballerina.toml versions is how the build script recognizes that it needs to release these

Comment (menakaj) [reply packages.json:1]: ok. will revert.

Comment (menakaj) [reply packages.json]: Reverted https://github.com/wso2/choreo-apim-mediation-policies/pull/19/commits/8ae68c3d97f0c5f204ead171fff9bd7879be4a33

**Review by pubudu91 — APPROVED** [review]

File: mediation-add-header/Package.md [hunk 2]
```diff
@@ -2,9 +2,8 @@
 
 ## Overview
 
-The "Add Header" policy is available out-of-the-box in Choreo and provides the capability to add arbitrary headers to either 
-the request or the response flowing through an HTTP proxy created in Choreo. Adding a header will append the specified value to the 
-list of values if the header already exists.
+This policy provides the capability to add arbitrary headers to either the request or the response. 
+Adding the same header multiple times will not overwrite the existing header values and it will be appended to the list of values if the header already exists.
```
Comment (shaniR) [inline mediation-add-header/Package.md:6]: Shall we bold the word - not?

Adding the same header multiple times will **not** overwrite the existing header and its values. When you add an existing header, it gets appended to the list of values. For example: <lets add an example>
 Better if we can mention a example.

File: mediation-set-header/Package.md [hunk 3]
```diff
@@ -1,8 +1,7 @@
 # Set Header
 
 ## Overview
-This package contains the mediation policy "Set Header" available out of the box in Choreo. This policy provides the capability to set 
-arbitrary headers to either the request or the response flowing through an HTTP proxy created in Choreo. 
+This policy provides the capability to set arbitrary headers to either the request or the response. Setting the same header multiple times will overwrite the existing header value.
```
Comment (shaniR) [inline mediation-set-header/Package.md:4]: It will overwrite? or not overwrite?

**Review by shaniR — CHANGES_REQUESTED** [review]

File: mediation-set-header/Package.md (hunk 3 above)
Comment (menakaj) [reply mediation-set-header/Package.md:4]: this will overwrite the header.

Comment (shaniR) [reply mediation-set-header/Package.md:4]: ack

**Review by shaniR — APPROVED** [review]

**Pull request merged**